server/src/main/java/com/hedera/block/server/verification/BlockVerificationStatus.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.block.server.verification;
+
+/**
+ * An enum representing the status of block verification.
+ */
+public enum BlockVerificationStatus {
+    /**
+     * The Block has been verified.
+     */
+    VERIFIED,
+    /**
+     * The Block failed verification, either due to an invalid signature or an invalid hash.
+     */
+    SIGNATURE_INVALID
```

Comment:
This implies that it's strictly signature issue, maybe add invalid hash enum or change name to something more ambiguous, as we are not sure at this point what is the root cause ?

Reply:
✅ Done